### ISSUE TITLE
APUSH: AP Practice Questions stimuli not appearing in Reference View

### ISSUE THREAD

**Issue by openstaxalina** [openstax/webview#2274]
@mwest-WWprod commented on [Mon Sep 16 2019](https://github.com/openstax/tutor/issues/602)

GLOBAL
Tutor APUSH
https://tutor-content.openstax.org/course/88

Stimuli are not appearing in AP Practice Questions in Student/Teacher Reference View. See below example in Chapter 2 Chapter Intro Essay (2.2). Our manuscript input is provided for reference. Note that the stimuli does appear in the Question Library.

Reference View
![multi-part_assessments_APUSH](https://user-images.githubusercontent.com/19212926/64976820-3817ad00-d880-11e9-9f26-cee4a0e1e5df.png)

Manuscript
![multi-part_assessments_APUSH MS](https://user-images.githubusercontent.com/19212926/64976848-45cd3280-d880-11e9-84cc-e110a2ed0ffc.JPG)

**Comment by openstaxalina** [comment]
Question ID is 17707 in exercises.openstax.org.

Question stimulus does not show when question is imported from exercises into CNX instance: (search "According to the excerpt from Roger Williams" on this page: https://content05.cnx.org/contents/CmIfJ6vh@34.1:ati9GAKm@13/2-2-%F0%9F%93%96-Chapter-2-Introductory-Essay-1607-1763)

**Comment by openstaxalina** [comment]
Agree this must be resolved for final product.  We are looking into this @mwest-WWprod.  The _earliest_ this would be resolved is 10/23.  I do not have developer feedback on this yet; it may be more complex and take longer to resolve.  

However, my understanding is that you are currently QA'ing these assessments in the Question Library, so this issue is **not** holding up your QA.  Is that correct?

**Comment by mwest-WWprod** [comment]
@openstaxalina we are QAing in the Question Library, but we are going to end up needing to QA the questions again in the reference view to make sure everything looks good there. The urgency though is around the next pilot in October (I'll confirm date with Alisha), and because we are leaving the AP Practice Questions in the reference view, we do need to get the pertinent issues resolved before then (or maybe we can put a workaround for the pilot in place). I'll put that info/timeline to our team and gather thoughts.

**Comment by openstaxalina** [comment]
@mwest-WWprod got it.   I'll make a note to discuss the scope of the October pilot on Friday's call.  That would help inform whether we could advise a content workaround for the specific content in the pilot.  Meantime, I'll talk with developers to get a rough sense of the workload involved in this issue.

**Comment by openstaxalina** [comment]
tl;dr: need to know where the code that pulls in exercises is coming from.  Alina to follow up with CE Tech / Ross.

Documenting team investigation:

Example **from Tutor** where stimulus displays properly:
![Screen Shot 2019-09-19 at 11 30 29 AM](https://user-images.githubusercontent.com/7863733/65262924-fb8ec000-dad0-11e9-9e4a-b6b1baa4b401.png)

Example **from Tutor** of a SINGLE part question displaying properly:
![Screen Shot 2019-09-19 at 11 31 37 AM](https://user-images.githubusercontent.com/7863733/65262978-1bbe7f00-dad1-11e9-98e4-dbdf9eb2fb59.png)

Interface of Exercises showing multi-part question editor.  The stimulus is called the "Intro" here:
![Screen Shot 2019-09-19 at 11 32 25 AM](https://user-images.githubusercontent.com/7863733/65263022-3a247a80-dad1-11e9-8cd1-02b08bf5f6b2.png)

**Comment by brittweinstein** [comment]
@brittweinstein put in SS Sprint

**Comment by openstaxalina** [comment]
Display code is located in webview.  May have been Phil S. - can check commits.  

There is also code in the old PDF pipeline.

**Comment by openstaxalina** [comment]
This code was likely written in 2016 or first half of 2017.  By Ross or Phil S.

**Comment by brittweinstein** [comment]
The problem is in  ./embeddables/exercise-template

**Comment by openstaxalina** [comment]
Some additional info that Ross provided: "should be able to add a conditional stanza at the top of the template to render the stimulus."

**Comment by Stackblocks** [comment]
Like the pets at the end of Homeward Bound, the stimuli have crested the hill into the backyard of their new APUSH home and we are glad to see them again after spending so much time worried about their safety.

![image.png](https://images.zenhubusercontent.com/5af1f4c24b5806bc2bd257f2/c368372c-218e-4654-be47-7ee9c3e3d0f6)

**Comment by openstaxalina** [comment]
confirmed seeing this on content05 (screenshot below).  @oscryan, could you please reimport the APUSH collection in tutor-content and confirm you are seeing the stimuli on AP Practice Questions?  Please post a screenshot so we could also double-check the styling.

<img width="851" alt="Screen Shot 2019-10-23 at 11 36 39 AM" src="https://user-images.githubusercontent.com/7863733/67414917-97f12a00-f589-11e9-8cb8-a39e1af2552e.png">

**Comment by oscryan** [comment]
Reimported, and I am... not seeing them in tutor, in browse the book. https://tutor-content.openstax.org/book/88/page/96996

The manifest has ap-practice under reading processing instructions, and the questions are there, but not the stimulus.
<img width="747" alt="Screen Shot 2019-10-23 at 12 54 24 PM" src="https://user-images.githubusercontent.com/1895432/67420451-4306e100-f594-11e9-9d95-c4c05d35589c.png">

**Comment by oscryan** [comment]
now it works https://tutor-content.openstax.org/book/88/page/97177

**Comment by openstaxalina** [comment]
The above appears related to caching, or the book not being fully reimported yet.  Ryan and I both confirmed the stimuli show at the link she pasted just above: 
<img width="778" alt="Screen Shot 2019-10-23 at 1 14 35 PM" src="https://user-images.githubusercontent.com/7863733/67421963-291acd80-f597-11e9-9938-68716d11995c.png">
